### ISSUE TITLE
Jinja not rendering for sources in 0.14.0

### ISSUE THREAD

**Issue by nydnarb** [fishtown-analytics/dbt#1619]
### Issue description and Results
While upgrading to 0.14.0, I noticed an issue with the documentation site. Column descriptions for sources were not having their Jinja rendered. For example, if my source YML looked like this
```
source:
  - name: foo
    description: "{{ doc('bar') }}"
    columns:
      - name: baz
        description: "{{ doc('qux') }}"
```
then on the documentation site I would see the string {{ doc('qux') }}. 

However, this works just fine in 0.13.0. Furthermore, model descriptions and column descriptions for models are being rendered just fine. 

Lastly, it appears this was an issue before and was previously fixed: https://github.com/fishtown-analytics/dbt/issues/1484

### System information
dbt versions 0.13.0 and 0.14.0
python version 2.7.10
Mac OS Mojave 10.14.5

### Steps to reproduce
Here are snippets from my project. Also, the previously linked issue has a great explanation of how to reproduce:
sources.yml
```
version: 2

sources:
  - name: marketing_attribution
    tables:
      - name: account_lndc_attribution
        description: "{{ doc('marketing_attribution__account_lndc_attribution__description') }}"
        columns:
          - name: person_id
            description: "{{ doc('marketing_attribution__account_lndc_attribution__person_id') }}"
            tests:
              - not_null
              - unique
          - name: person_added_timestamp
            description: "{{ doc('marketing_attribution__account_lndc_attribution__person_added_timestamp') }}"
            tests:
              - not_null
```

sources.md
```
{% docs marketing_attribution__account_lndc_attribution__description %} Morbi in ipsum sit amet pede facilisis laoreet. Donec lacus nunc, viverra nec, blandit vel, egestas et, augue. Vestibulum tincidunt malesuada tellus. Ut ultrices ultrices enim. Curabitur sit amet mauris. Morbi in dui quis est pulvinar ullamcorper. {% enddocs %}

{% docs marketing_attribution__account_lndc_attribution__person_id %} Suspendisse potenti. Nunc feugiat mi a tellus consequat imperdiet. Vestibulum sapien. Proin quam. Etiam ultrices. Suspendisse in justo eu magna luctus suscipit. {% enddocs %}

{% docs marketing_attribution__account_lndc_attribution__person_added_timestamp %} Vestibulum lacinia arcu eget nulla. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Curabitur sodales ligula in libero. Sed dignissim lacinia nunc. Curabitur tortor. Pellentesque nibh. Aenean quam. In scelerisque sem at dolor. {% enddocs %}
```

Then, `dbt docs generate && dbt docs serve`.

## EDIT
The issue only affects source columns. A previous version of this issue reported that it was an issue with the table level description as well.

**Comment by drewbanin** [comment]
Thanks @nydnarb - we'll check this one out!

**Comment by drewbanin** [comment]
I can reproduce this specifically for column-level descriptions. Are you seeing that dbt fails to interpolate these descriptions specifically for columns, or are you seeing this same error for source-level / table-level descriptions?

**Comment by nydnarb** [comment]
Good catch! Yes, the description at the table level is fine. I'm only seeing the issue at column level.

**Comment by drewbanin** [comment]
Fixed in #1633 -- this is going out in 0.14.1
